docs/settings/alert-action-settings.asciidoc

```diff
@@ -71,6 +71,13 @@ a|`xpack.actions.`
   +
   As an alternative to setting both `xpack.actions.proxyRejectUnauthorizedCertificates` and `xpack.actions.rejectUnauthorized`, you can point the OS level environment variable `NODE_EXTRA_CA_CERTS` to a file that contains the root CAs needed to trust certificates.
 
+| `xpack.actions.maxResponseContentLength` {ess-icon}
+  | Defines the max size of the http response content in bytes allowed in node.js. Defaults to 1000000 (1MB).
+
+| `xpack.actions.responseTimeout` {ess-icon}
+  | Specifies the time before the request times out. If the request takes longer than `xpack.actions.responseTimeout`, the request will be aborted. Defaults to 60s.
```

Comment:
Similar on specificity of what this config will be applied to:

```suggestion
| `xpack.actions.responseTimeout` {ess-icon}
  | Specifies the maximum time allowed for requests to external resources by actions. If a request takes longer to complete than the specified duration it will be aborted. This time is formatted as a duration of `<count>[ms|s|m|h|d|w|M|Y]` (e.g. '20m', '24h', '7d', '1w'). Defaults to `60s`.
```

Reply:
@gchaps I suspect I'm being too verbose here (as usual). Would love your thoughts.

Reply:
> Specifies the time allowed for requests to external resources. Requests that take longer are aborted. The time is formatted as `<count>[ms|s|m|h|d|w|M|Y]`, for example, '20m', '24h', '7d', '1w'. Defaults to `60s`.